test/unit/lib/executor/driver-handler-tests.js

```diff
@@ -145,30 +145,128 @@ describe('lib/executor/driver-handler.js', function() {
             delete process.env.COMMIT_HASH;
             delete process.env.NODE_ENV;
             delete process.env.TUNNEL_IDENTIFIER;
-            delete process.env.SAUCE_CONFIG;
+            delete process.env.SAUCE_CAPABILITIES;
             mockery.resetCache();
             mockery.deregisterAll();
             mockery.disable();
         });
 
-        describe('if process.env.SAUCE_CONFIG is set', function() {
+        describe('if process.env.SAUCE_CAPABILITIES is set', function() {
             it('should call webdriver.withCapabailities once with the' +
-                'capabilities as the JSON.parsed process.env.SAUCE_CONFIG', function() {
-                process.env.SAUCE_CONFIG =`{"mySauceConfig": "myConfig"}`;
+                'capabilities as the JSON.parsed process.env.SAUCE_CAPABILITIES', function() {
+                process.env.SAUCE_CAPABILITIES =`{"mySauceConfig": "myConfig"}`;
 
                 driverHandler.inSaucelabs();
 
                 expect(webdriverBuilder.withCapabilities.args).to.deep.equal([
                     [
                         {
-                            mySauceConfig: 'myConfig',
+                            'accessKey': 'sauceAccessKey',
+                            'browserName': 'chrome',
+                            'customData': {
+                            'build': 'buildNumber',
+                            'commithash': 'commitHash',
+                            'environment': 'test',
+                            'release': 'releaseVersion',
+                            },
+                            'mySauceConfig': 'myConfig',
+                            'name': 'my-test.json',
+                            'platform': 'Windows 10',
+                            'tunnel-identifier': 'mbtt-tunnel',
+                            'username': 'sauceUsername',
+                            'version': '63.0',
                         },
                     ],
                 ]);
             });
+            describe('if the field in the sauce config contains a sub-object', function() {
```

Comment:
There needs to be a seperate unit test describe block for each function on the exported object.

Reply:
Done

Reply:
I believe there the `_navigateToAndModifyObject` still needs to mocked out here.